src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1601,7 +1660,9 @@ void WholeBodyDynamicsDevice::publishEstimatedQuantities()
             //publishFilteredInertialForGravityCompensator();
 
             //Send filtered force torque sensor measurment, if requested
-            //publishFilteredFTWithoutOffset();
+            if( streamFilteredFT){
+            publishFilteredFTWithoutOffset();
```

Comment:
Please indent the code consistently.

Reply:
Done in a83c188a98e2bd4f75f58a484361ed2a88fffea7